sdk/src/test/java/io/dapr/actors/runtime/DefaultActorFactoryTest.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ */
+
+package io.dapr.actors.runtime;
+
+import io.dapr.actors.ActorId;
+import org.junit.Assert;
+import org.junit.Test;
+
+import static org.mockito.Mockito.*;
+
+/**
+ * Testing the default constructor of an Actor.
+ */
+public class DefaultActorFactoryTest {
+
+  /**
+   * A compliant implementation of Actor to be used in the tests below.
+   */
+  static class MyActor extends AbstractActor implements Actor {
+
+    ActorService actorService;
+
+    ActorId actorId;
+
+    public MyActor(ActorService actorService, ActorId actorId) {
+      this.actorService = actorService;
+      this.actorId = actorId;
+    }
+  }
+
+  /**
+   * A non-compliant implementation of Actor to be used in the tests below.

```

Comment:
"non-compliant" is usually written as a single word.

Reply:
As per the dictionary in IntelliJ, "non-compliant" is correct and "noncompliant" is wrong. My browser also complains about "noncompliant".